src/Prelude.purs

```diff
@@ -314,9 +270,16 @@ liftA1 f a = pure f <*> a
 -- |    y <- m2 x
 -- |    m3 x y
 -- | ```
-class (Apply m) <= Bind m where
+class Apply m <= Bind m where
   bind :: forall a b. m a -> (a -> m b) -> m b
 
+infixl 1 bind as >>=
+
+bindFlipped :: forall m a b. Bind m => (a -> m b) -> m a -> m b
```

Comment:
```
-- | `bindFlipped` is `bind` with its arguments reversed. For example:
-- |
-- | ```purescript
-- | print =<< random
-- | ```
```

Does the fixity here allow you to do things like `f <<< g <<= xs`?